api/enterprise-api-ref/first-time-setup.markdown

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Fist Time Setup API
```

Comment:
```suggestion
title: Fist time setup API
```

Reply:
@nickanderson @olehermanse don't we want this change? https://github.com/cfengine/documentation/?tab=readme-ov-file#capitalization